app/src/main/java/it/niedermann/owncloud/notes/android/activity/NotesListViewActivity.java

```diff
@@ -347,6 +352,10 @@ public void onItemClick(NavigationAdapter.NavigationItem item) {
                 } else if (item == itemAbout) {
                     Intent aboutIntent = new Intent(getApplicationContext(), AboutActivity.class);
                     startActivityForResult(aboutIntent, about);
+                } else if (item == itemTrashbin) {
+                    SharedPreferences preferences = PreferenceManager.getDefaultSharedPreferences(getApplicationContext());
+                    preferences.getString(SettingsActivity.SETTINGS_URL, SettingsActivity.DEFAULT_SETTINGS);
```

Comment:
This is doubled and should be removed.

Reply:
Fixed in 0762f9825e3a1fa9c2dad6464386c7b686778bec